index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- CSS stylesheet -->
+    <link rel="stylesheet" href="styles.css">
+    <!-- Fonts links -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!-- HEADER -->
+    <header class="header">
+        <div class="logo"><p>My Logo</p></div>
+        <!-- naviguation bar -->
+        <nav class="menu-box">
+            <label for="menu"><input type="checkbox" name="menu" id="menu" class="menu"></label>
+            <ul class="menu-list">
+                <li class="menu-item">Portfolio</li>
+                <li class="menu-item">About</li>
+                <li class="menu-item">Contact</li>
+            </ul>
+        </nav>
+    </header>
+    <!-- HEADER (end) -->
+    <!-- HEADLINE -->
+    <section class="headline">
+        <div class="headline-content">
+            <h1>Hey There.<br>I'm Nelly</h1>
+            <p>I'm a Software Developer</p>
+            <p>
+                I can help you build a product , feature or website. 
+                Look through some of my work and experience! If you 
+                like what you see and have a project you need coded, 
+                don’t hesitate to contact me.
+            </p>
+        </div>
+        <div class="socials">
+            <ul>
+                <li><img src="images/github.png" alt="twitter icon"></li>
+                <li><img src="images/Linkedin icon.png" alt="Linkedin icon"></li>
+                <li><img src="images/angel-list.png" alt="twitter icon"></li>
+                <li><img src="images/twitter.png" alt="twitter icon"></li>
+                <li><img src="images/medium.png" alt="twitter icon"></li>
```

Comment:
- [ ] Each image in the list item should be wrapped inside an `a` tag. Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href.